<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CogWheelServerSettingsDialog</class>
 <widget class="QDialog" name="CogWheelServerSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>413</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Server Settings</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>350</y>
     <width>203</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QPushButton" name="cancelButton">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="saveButton">
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>0</y>
     <width>351</width>
     <height>337</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QCheckBox" name="serverEnabled">
      <property name="text">
       <string>enabled</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="labelServerName">
          <property name="text">
           <string>Server Name:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="serverName"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="labelServerPort">
          <property name="text">
           <string>Server Port:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="serverPort"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="labelServerVerion">
          <property name="text">
           <string>Server Version:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="serverVersion"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <widget class="QCheckBox" name="serverAllowSMNT">
      <property name="text">
       <string>Allow SMNT</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QCheckBox" name="serverTlsSslEnabled">
      <property name="text">
       <string>TLS/SSL</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_14">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Server Key File:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="serverKeyFile"/>
      </item>
     </layout>
    </item>
    <item row="4" column="0">
     <widget class="QCheckBox" name="serverPlainFTPEnabled">
      <property name="text">
       <string>Plain FTP</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QCheckBox" name="serverAnonymousEnabled">
      <property name="text">
       <string>Anonymous Login</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Server Cert File: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="serverCertFile"/>
      </item>
     </layout>
    </item>
    <item row="6" column="0">
     <widget class="QCheckBox" name="serverLoggingEnabled">
      <property name="text">
       <string>Logging Enabled</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
